database/mssql/test/versions/v_35_1_test.sql

```diff
@@ -1,7 +1,7 @@
 -- Test that the auth groups have been inserted correctly
 SET NOCOUNT ON
 
-SELECT COUNT(*) FROM $(DB_NAME).[access].[ORBC_USER_AUTH_GROUP_TYPE] 
+SELECT COUNT(*) FROM $(DB_NAME).[access].[ORBC_GROUP_ROLE] 
```

Comment:
Does the main test 35.sh file needs to be updated?

Reply:
nope, just this one.